src/View/View.php

```diff
@@ -1348,7 +1348,7 @@ protected function _getTemplateFileName(?string $name = null): string
         $name ??= $this->template;
 
         if (!$name) {
-            throw new CakeException('Template name not provided');
+            throw new MissingTemplateException('Template name not provided');
```

Comment:
`MissingTemplateException` generate error message of a specific format. This will end up generating a message like 
```
Template file `Template name not provided` could not be found.
```